src/components/VukLogo/VukLogo.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <svg
+    :style="{
+      height: height,
+      width: width
+    }"
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 652.86 164.14"
+  >
+    <!-- Path representing the text of the VATUK logo -->
+    <path
+      d="M19.34,50.81,30,84c1.88,5.78,3.22,11,4.43,16.39h.41C36,95,37.47,89.89,39.22,84L49.56,50.81H67L42,116.22H25.79L1.48,50.81Zm93.75,65.41L112,109h-.4c-4,5.11-10.88,8.73-19.34,8.73-13.16,0-20.55-9.53-20.55-19.47,0-16.52,14.64-24.85,38.81-24.71V72.43c0-4.29-1.74-11.41-13.29-11.41a34.68,34.68,0,0,0-17.6,4.83L76.42,55.11c4.84-3,13.3-5.78,23.64-5.78,21,0,27,13.3,27,27.53V100.5a105.18,105.18,0,0,0,.94,15.72Zm-2.15-32C99.25,84,88.11,86.54,88.11,96.47c0,6.45,4.16,9.41,9.4,9.41a13.38,13.38,0,0,0,12.89-9,12,12,0,0,0,.54-3.63Zm51.71-52.11V50.81h15.71V63.17H162.65V92c0,7.93,2.14,12.09,8.46,12.09a24,24,0,0,0,6.44-.81l.27,12.63c-2.41.94-6.71,1.61-11.95,1.61-6,0-11.15-2-14.24-5.37-3.49-3.76-5.1-9.67-5.1-18.27V63.17h-9.4V50.81h9.4V36ZM190,101a36.89,36.89,0,0,0,16.79,4.7c7.26,0,10.48-2.95,10.48-7.25s-2.69-6.72-10.75-9.54c-12.76-4.43-18.13-11.41-18-19.07,0-11.55,9.53-20.55,24.71-20.55a38.39,38.39,0,0,1,17.33,3.9l-3.23,11.68a29.39,29.39,0,0,0-13.83-3.76c-5.91,0-9.14,2.82-9.14,6.85s3.09,6.18,11.42,9.13c11.82,4.3,17.33,10.35,17.46,20,0,11.81-9.27,20.41-26.59,20.41-7.93,0-15-1.88-19.88-4.57Zm74.41-68.5c0,5-3.63,8.87-9.4,8.87-5.51,0-9.14-3.9-9.14-8.87s3.77-9,9.27-9A8.83,8.83,0,0,1,264.45,32.54Zm-17.6,83.68V50.81h16.66v65.41Zm34.39-45.94c0-7.52-.14-13.83-.54-19.47h14.24l.67,9.67h.4c3.22-5.1,9.13-11.15,20.15-11.15a18.86,18.86,0,0,1,18.13,12.09h.27a24.68,24.68,0,0,1,8.06-8.19,21.84,21.84,0,0,1,13.16-3.9c10.88,0,21.89,7.39,21.89,28.34v38.55H361.55V80.09c0-10.88-3.76-17.33-11.68-17.33-5.64,0-9.81,4-11.55,8.73a23.3,23.3,0,0,0-.81,5.51v39.22H321.4V78.34c0-9.13-3.63-15.58-11.29-15.58-6.17,0-10.34,4.84-11.82,9.4a14.24,14.24,0,0,0-.94,5.38v38.68H281.24Zm201,26.33c0,7.79.26,14.23.53,19.61h-14.5l-.81-9.94h-.27c-2.82,4.7-9.27,11.41-20.95,11.41-11.95,0-22.83-7.11-22.83-28.47V50.81h16.52V86.4c0,10.88,3.49,17.86,12.22,17.86,6.58,0,10.88-4.7,12.62-8.86a14.94,14.94,0,0,0,1-5.1V50.81h16.52Zm34.38-17.06h.27c1.61-2.55,3.49-5.37,5.23-7.66L538,50.81h19.88L533.67,77.54l27.67,38.68H541.06l-18.8-28.88-5.64,6.58v22.3H500.1V20.86h16.52Z"
+      :class="textColor"
+      class="fill-current"
+    />
+    <!-- Path repreenting the tail of the VATUK logo -->
+    <path
+      d="M495.6,153.39a10.75,10.75,0,1,1,10.75,10.75A10.75,10.75,0,0,1,495.6,153.39Zm57.75,2.09a10.75,10.75,0,1,0-10.75-10.75A10.75,10.75,0,0,0,553.35,155.48Zm40.34-25.26a10.75,10.75,0,1,0-10.75-10.75A10.76,10.76,0,0,0,593.69,130.22Zm27.66-40A10.75,10.75,0,1,0,610.6,79.48,10.75,10.75,0,0,0,621.35,90.23Zm31.51-68.61L631.24,0,609.62,21.62l21.62,21.62Z"
+      :class="tailColor"
+      class="fill-current"
+    />
+  </svg>
+</template>
+
+<script lang="ts">
+  import Vue from 'vue'
+  import { Component, Prop, Ref } from 'vue-property-decorator'
+  import { TailColors } from './logoVariants'
```

Comment:
I can't see this being used anywhere - All of the validation of variants seems to be inline?

Reply:
Removed - good catch!